qa/L0_metrics/test.sh

```diff
@@ -171,23 +171,31 @@ for metric in ${CPU_METRICS}; do
     prev_value=`curl -s localhost:8002/metrics | grep ${metric} | grep -v "HELP\|TYPE" | awk '{print $2}'`
     echo "  Initial ${metric} value: ${prev_value}"
 
+    num_not_updated=0
     for (( i = 0; i < $num_iterations; ++i )); do
       sleep $WAIT_INTERVAL_SECS
       current_value=`curl -s localhost:8002/metrics | grep ${metric} | grep -v "HELP\|TYPE" | awk '{print $2}'`
       echo "  ${metric} value: ${current_value}"
       if [ $current_value == $prev_value ]; then
+        num_not_updated=$((num_not_updated+1))
+      fi
+      prev_value=$current_value
+    done
+
+    # Give CPU metrics some tolerance to not update
+    # An alternative may be to run some busy work in the background
```

Comment:
You should describe the new logic in L165-L166. Add the alternative for busy work as a TODO and create a jira ticket for the same.

Reply:
I wasn't sure if there was any benefit to introducing busy work or not, moreso just offering a future alternative if this tolerance wasn't OK - so I didn't want to add a JIRA ticket for something we may never do. Do you think it's worth adding?

Reply:
If the test keeps failing then it would be a nice add to ensure the test robustness.

Reply:
added ticket, added threshold variable, updated comments